code/game/objects/items.dm

```diff
@@ -211,6 +211,13 @@ GLOBAL_DATUM_INIT(fire_overlay, /mutable_appearance, mutable_appearance('icons/e
 	/// Used in obj/item/examine to determines whether or not to detail an item's statistics even if it does not meet the force requirements
 	var/override_notes = FALSE
 
+	/// the name of the last suicider oo spooky
```

Comment:
This should probably be a component and only for certain items.  I'd recommend you look at the toy mech suicide code and try to c/p that as a component.